packages/styles/forms.css

```
@@ -94,10 +95,9 @@ textarea:focus,
 [role='listbox']:focus,
 [role='spinbutton']:focus {
   outline: 0;
-  border: 1px solid var(--field-border-color-focus);
-  box-shadow: 0 0 0 2px var(--field-border-color-focus),
-    0 0 5px var(--field-border-color-focus-glow),
-    inset 0 1px 2px rgba(0, 0, 0, 0.05);
+  border: 2px solid var(--field-border-color-focus);
+  box-shadow: 0px 0px 5px var(--field-border-color-focus-glow),
+    inset 0px 1px 2px rgba(0, 0, 0, 0.05);
```

So the trick we had before with the focus ring to not displace content was a combination of:

- 1px outline
- 2px outer solid box-shadow (+ focus glow)

To shrink the solid outline while keeping the same focus footprint, we just need to reduce the box-shadow by 1px:

```suggestion
  border: 1px solid var(--field-border-color-focus);
  box-shadow: 0 0 0 1px var(--field-border-color-focus),
    0 0 5px var(--field-border-color-focus-glow),
    inset 0px 1px 2px rgba(0, 0, 0, 0.05)
```

This will need to be applied to both `TextField` and `Combobox`